pkg/builder/options.go

```diff
@@ -138,3 +138,19 @@ var (
 )
 
 // }}}
+
+// MatchEveryOwner determines whether the watch should be filtered based on
+// controller ownership. As in, when the OwnerReference.Controller field is set.
+//
+// If set to true, the handler receives notification for every owner of the object
```

Comment:
Nit: it's more like "if set" and "if not set", right? You can't actually set it to true and false (the user-visible option)